### ISSUE TITLE
Verwaltung von Facebook-Seiten im Betreiberbereich

### ISSUE THREAD

**Issue by Bensk1** [Strassengezwitscher/Strassengezwitscher#82]
Veranstaltungen müssen in einer Liste angezeigt, bearbeitet, redaktionell geprüft und freigeschaltet werden können. 

Ziemlich analog dazu laufen die Veranstaltungen.

**Comment by mathebox** [comment]
Bitte genauer spezifizieren!

Zurzeit ist es werden schon Facebook-Seiten und Veranstaltungen in Listen angezeigt und können auch bearbeitet werden. Inwieweit wir die redaktionell Prüfung unterstützen können müsste geklärt werden. Beide können auch über das Bearbeitungsformular freigeschaltet (active = True) werden.

**Comment by Bensk1** [comment]
Ja, das EPIC ist work in progress, ich kommuniziere momentan noch mit Alexej.

**Comment by piepmatz** [comment]
Gibts hier was Neues?

**Comment by Bensk1** [comment]
Funktioniert alles, müssten nur noch die Workflows etwas optimieren.
